cmake/projects/WebP/hunter.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018, Ruslan Baratov
+# Copyright (c) 2018, Mathieu-Andre Chiasson
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_configuration_types)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+        PACKAGE_NAME
+        WebP
+        VERSION
+        "0.6.1-p0"
+        URL
+        "https://github.com/mchiasson/libwebp/archive/v0.6.1-p0.tar.gz"
```

Comment:
FYI, this url is wrong, and will need to be changed to `hunter-package/libweb` 

https://github.com/hunter-packages/libwebp/pull/1